CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- Added `transformer_kwargs` argument to `PretrainedTransformerBackbone`
```

Comment:
This actually needs to move up above line 11

Reply:
Sorry, got confused because there wasn't an "Unreleased" section before.

Reply:
My bad, I snuck that in 😏